lib/src/descriptor.pb.dart

```diff
@@ -204,7 +207,8 @@ class _ReadonlyDescriptorProto_ExtensionRange
     extends DescriptorProto_ExtensionRange with ReadonlyMessageMixin {}
 
 class DescriptorProto_ReservedRange extends GeneratedMessage {
-  static final BuilderInfo _i = new BuilderInfo('DescriptorProto_ReservedRange')
+  static final BuilderInfo _i = new BuilderInfo('DescriptorProto_ReservedRange',
+      packageName: 'google.protobuf.DescriptorProto')
```

Comment:
Is it correct that nested proto messages share the same `packageName`?
```
 91 // Describes a message type.
 92 message DescriptorProto {
 ...
101   message ExtensionRange {
102     optional int32 start = 1;
103     optional int32 end = 2;
104   }
105   repeated ExtensionRange extension_range = 5;
106 
114   message ReservedRange {
115     optional int32 start = 1; // Inclusive.
116     optional int32 end = 2;   // Exclusive.
117   }
118   repeated ReservedRange reserved_range = 9;
122 }
```

Turns into
```
 class DescriptorProto_ExtensionRange extends GeneratedMessage {
  static final BuilderInfo _i = new BuilderInfo(
      'DescriptorProto_ExtensionRange',
      packageName: 'google.protobuf.DescriptorProto') 
    ...
}
 class DescriptorProto_ReservedRange extends GeneratedMessage {
  static final BuilderInfo _i = new BuilderInfo('DescriptorProto_ReservedRange',
      packageName: 'google.protobuf.DescriptorProto');
}
```

Notice that both classes have the same `packageName`.